resize-observer-1/Overview.bs

```diff
@@ -135,14 +137,25 @@ It is modeled after {{MutationObserver}} and {{IntersectionObserver}}.
 
 <pre class="idl">
     enum ResizeObserverBoxOptions {
-        "border-box", "content-box"
+        "border-box", "content-box", "device-pixel-content-box"
     };
 </pre>
 
 ResizeObserver can observe different kinds of CSS sizes:
 
 * {{border-box}}  : size of <a>box border area</a> as defined in CSS2.
 * {{content-box}} : size of <a>content area</a> as defined in CSS2.
+* {{device-pixel-content-box}} : size of <a>content area</a> as defined in CSS2, in device pixels,
+before applying any CSS transforms on the element or its ancestors.
+This size must contain integer values.
+
+<p class="note">
```

Comment:
How about adding a non-normative note specifying a typical way that UAs compute this box?

e.g. "How a UA computes the device pixel box for an element is implementation-dependent. One way could be to multiply the box size and position by the device pixel ratio, then round the resulting (floating-point) size and position of the box to integer values in some way that maximizes quality of rendered output".